site/src/pages/ManagementSettingsPage/OrganizationSummaryPageView.tsx

```diff
@@ -0,0 +1,50 @@
+import type { FC } from "react";
+import type { Organization } from "api/typesGenerated";
+import {
+  PageHeader,
+  PageHeaderTitle,
+  PageHeaderSubtitle,
+} from "components/PageHeader/PageHeader";
+import { Stack } from "components/Stack/Stack";
+import { UserAvatar } from "components/UserAvatar/UserAvatar";
+
+interface OrganizationSummaryPageViewProps {
+  organization: Organization;
+}
+
+export const OrganizationSummaryPageView: FC<
+  OrganizationSummaryPageViewProps
+> = (props) => {
```

Comment:
similar to my comment on your other PR, we use destructuring for props almost everywhere in the frontend.

```suggestion
> = ({ organization }) => {
```